src/core/init/load_on_media_source.ts

```diff
@@ -174,6 +174,14 @@ export default function createMediaSourceLoader({
               handleDiscontinuity(gap[1], mediaElement);
             }
             return EMPTY;
+          case "needs-source-buffer-flush":
+            if (mediaElement.currentTime + 0.001 < mediaElement.duration) {
```

Comment:
Does this seek operation works on all platforms? Or is it something that is unique to Chrome, Firefox, etc?

Reply:
I don't know.

It also works on some set-top boxes which are CEF-based